website/static/js/nodeControl.js

```diff
@@ -37,12 +37,12 @@ var MESSAGES = {
                         'but will have no impact on external sites, including Google\'s cache. ' +
                         'Would you like to continue?',
     // TODO(hrybacki): Remove once Retraction/Embargoes goes is merged into production
-    makeRegistrationPublicWarning: '<b>Important Note:</b> Effective <u>June 1, 2015</u>, new registrations ' +
+    makeRegistrationPublicWarning: '<b>Important Note:</b> Effective <u>June 8, 2015</u>, new registrations ' +
                         'will be made public immediately or can be embargoed for up to one year. There ' +
                         'will no longer be the option of creating a permanently private registration. This ' +
-                        'registration occurred before June 1, 2015, so you do retain the option of keeping it ' +
+                        'registration occurred before June 8, 2015, so you do retain the option of keeping it ' +
                         'private. However, if you do choose to make the registration public now, then after ' +
-                        'June 1, 2015 you will not be able to return it to private. Are you sure that you would like ' +
+                        'June 8, 2015 you will not be able to return it to private. Are you sure that you would like ' +
```

Comment:
Were these date changes meant for a different pull request? Usually it's best to only include commits relevant to a single feature/bug fix in a pull request so that they can be evaluated separately.